ui/src/components/payment/payment-dialog/PaymentDialog.tsx

```diff
@@ -305,10 +338,10 @@ export default class PaymentDialog extends Component<PaymentProps, PaymentState>
         const studentSurnameTf = this.formTextField(surname, "student-surname", "Surname", this.getErrorMessageIfFieldEmpty, this.changeSurname);
```

Comment:
Maybe better to remove `If` ending?

Reply:
its TextField mark, ill guess new naming